src/views/VenueDetails/BookVenue/index.jsx

```diff
@@ -37,14 +37,14 @@ export function BookVenue ({ venueDetails }) {
     if (!value[1]) {
       secondDayTime = firstDayTime
     }
-    amountOfBookedDays = Math.round((Math.abs((firstDayTime - secondDayTime) / (dayInMilliseconds))) + 1)
+    amountOfBookedDays = Math.round((Math.abs((firstDayTime - secondDayTime) / (DAY_IN_MILLISECONDS))) + 1)
   }
 
   useEffect(() => {
     fetch('https://api.apilayer.com/exchangerates_data/convert?to=PLN&from=EUR&amount=1',
       {
         headers: {
-          apikey: 'DtbpehV28fDVm4s3sLtJl3vWCAcH8cZU'
+          apikey: 'NrQIQreA18mCUi76aT6IACgTitJMd8u4'
```

Comment:
Move this kind of sensitive informations out of the code. Create `.env` file and then use `process.env.VARIABLE_NAME` 
https://create-react-app.dev/docs/adding-custom-environment-variables/